lib/smart_proxy_host_reports/ansible_processor.rb

```diff
@@ -61,4 +71,49 @@ def spool_report
     end
     SpooledHttpClient.instance.spool(report_id, payload)
   end
+
+  private
+
+  def log_level(log)
+    if log["failed"]
+      "err"
+    elsif log["changed"]
+      "notice"
+    else
+      "info"
+    end
+  end
+
+  def log_source(log)
+    # Temporary until  is resolved
```

Comment:
*until https://github.com/theforeman/foreman-ansible-modules/pull/1267#discussion_r711760023 is resolved.